src/com/mambu/apisdk/model/ScheduleQueryParam.java

```diff
@@ -0,0 +1,24 @@
+package com.mambu.apisdk.model;
+
+/**
+ * Shows possible values of schedule query parameters
+ * 
+ * @author acostros
+ */
+public enum ScheduleQueryParam {
```

Comment:
It looks like we also have some other parameters for repayment schedule, such as principalRepaymentInterval or repaymentPeriodCount. Is there any reason why we do not put the newly added params from this task in the same place with the other (old) ones?

Reply:
added description on ScheduleQueryParams with the other params (loan related) that can be set

Reply:
Thank you!